source/test/runbook_validator.test.ts

```diff
@@ -145,7 +142,7 @@ function getControlRunbooks(runbooks: RunbookTestHelper[]): RunbookTestHelper[]
 }
 
 function getRemediationRunbooks(runbooks: RunbookTestHelper[]): RunbookTestHelper[] {
```

Comment:
Same here, could  `runbooks.filter(it => !it.isRemediationRunbook())` in line 180 and remove this function.